docs/prow/prow-cluster-update.md

```diff
@@ -0,0 +1,49 @@
+# Prow cluster update
+
+In order to get improved/newer versions of prow we need to update prow. The kubernetes teams' prow instance is getting updated via a shell script. This shell script offers a small list of the last pushed container tags. This could reflect only containers of a single day, due to how many times the kubernetes team is actually running their pipeline. This limits us in the versions we can choose from, as we usually don't go from finding out about a new version, to deploying it, within one day. To cherrypick updates, we will need a different process. By monitoring [prow announcements](https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md) we can see when fixes or important changes are merged into the kubernetes teams' repository and start looking into applying these changes to our prow cluster. This document aims to provide a description of how to perform such an update.
+
+## Update process
+
+To update a Prow cluster follow these steps:
+
+1. Follow [this](./prow-installation-on-forks.md) document for details to set up a Prow cluster.
+2. Go to the [`kubernetes/test-infra`](https://github.com/kubernetes/test-infra/) project and select a commit with the desired update for the Prow cluster. For example, use [2c8e0dbb96b4c1a86d42275dfbed5474a6d05def](https://github.com/kubernetes/test-infra/commit/2c8e0dbb96b4c1a86d42275dfbed5474a6d05def).
+3. Open both [`starter.yaml`](../../prow/cluster/starter.yaml) in the current project and [`starter.yaml`](https://github.com/kubernetes/test-infra/blob/2c8e0dbb96b4c1a86d42275dfbed5474a6d05def/prow/cluster/starter.yaml) in the Kubernetes project and copy new tags for these containers:
+    * gcr.io/k8s-prow/hook
+    * gcr.io/k8s-prow/plank
+    * gcr.io/k8s-prow/sinker
+    * gcr.io/k8s-prow/deck
+    * gcr.io/k8s-prow/horologium
+    * gcr.io/k8s-prow/tide
+4. Copy the commit ID into a comment at the top of the file to keep track of the current release used for the deployments.
+5. Check with your preferred diff tool for other meaningful changes (e.g. stability update in a prow component or additional configurations on existing components) that need to be taken into account and copied over.
```

Comment:
I'm still not sure about the word "meaningful". Who decides if a change is meaningful or not? It is unclear for a reader what they should copy. Should they look for more changes between the `starter.yaml` files or check other files? Is 7th step of the instruction a duplication or does it refer to the `config.yaml` files?

Reply:
I'd assume that SIG-Deploy-CI decides on meaningful. So far, the upgrade progress includes only starter.yaml and config.yaml.

5th step applies to starter.yaml, 7th step applies to config.yaml

Reply:
```suggestion
5. Use your preferred diff tool to check and copy the stability update in a Prow component and additional configurations on existing components.
```